cyoa/app/views/rounds/marvel_characters_index.html.haml

```diff
@@ -0,0 +1,12 @@
+.row
+  .small-6.large-centered.columns
+    %h2 WHAT IS THE NAME OF THIS CHARACTER?
+    %p
+      .center-image
+        %img{src: @correct_question.trivia_piece}
+      - @questions.shuffle.each do |question|
+        =form_for @current_round, data: {round_number: 'cats'} do |f|
+          =f.hidden_field :answer, value: question.id
+          =f.submit value: question.name, data: {submit_answer: 'blah'}, class: "character-button button"
+
+    %div{data:{result_holder: 'display'}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace